src/pudl/metadata/resources/eia860m.py

```diff
@@ -32,6 +32,7 @@
                 "energy_source_code_1",
                 "energy_storage_capacity_mwh",
                 "fuel_type_code_pudl",
+                "generator_operating_date",
```

Comment:
Given that this is the only change, I'm assuming that the column is present in the dataframe that generates this asset, and it was getting dropped when we enforce the schema?